custom_components/tuya_local/devices/outdoor_inc_zipblind.yaml

```diff
@@ -0,0 +1,56 @@
+name: Outdoor Zip Blind
+products:
+  - id: f91miyhj
+    manufacturer: Outdoor Inc
+    model: DC1545BE
+entities:
+  - entity: cover
+    class: blind
+    current_position_dp: 8
+    set_position_dp: 9
```

Comment:
These two are not valid syntax. They are defined properly in the dps list below, so can be removed